scripts/travis-test.sh

```diff
@@ -2,6 +2,9 @@
 
 set -eu
 
+# Tune the GC to use more memory to reduce the number of garbage collections
+export GOGC=400
```

Comment:
do we need this as well as DENDRITE_LINT_DISABLE_GC ?

Reply:
`DENDRITE_LINT_DISABLE_GC` won't disable GC for the unit/integration tests or anything, which I guess is why its there?